deploy/internal/job-upgrade-db.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: db-migrate
+  labels:
+    app: noobaa
+spec:
+  backoffLimit: 5
+  activeDeadlineSeconds: 1200
```

Comment:
I was thinking about a job being stuck for some reason - setting a 20 minutes cap sounded reasonable

Reply:
until we'll do more runs and see the behavior I prefer to not set it. this can also mask some problems if the migration is stuck or taking a long time.

in general we should decide what to do in case of migration failure

Reply:
set to default